### PR TITLE
Spell 'ingestor' as 'ingester'

### PR DESCRIPTION
**Pull request by jml**
This change is [Reviewable](https://reviewable.io/reviews/tomwilkie/prometheus/56)

### CODE REVIEW

File: frankenstein/cmd/frankenstein/main.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func main() {
 		maxChunkAge          time.Duration
 	)
 
-	flag.StringVar(&mode, "mode", distributor, "Mode (distributor, ingestor).")
```
Comment (tomwilkie) [inline frankenstein/cmd/frankenstein/main.go:64]: This will need an update in service-conf.

**Comment by tomwilkie** [comment]
LGTM assuming its green.

**Comment by juliusv** [comment]
:+1: Yayyy! But yeah, service-conf stuff of course needs equivalent updates.

**Comment by jml** [comment]
https://github.com/weaveworks/service-conf/pull/149

**Pull request merged**